docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -234,7 +234,7 @@ For `logstash` the following arguments are supported:
   * `args.password`: (Optional) Password of the {ls} instance to connect to. Mandatory if HTTP Basic authentication is enabled in {ls}.
   * `args.max_batch_size`: (Optional) Maximum number of events to send in a single HTTP(s) request. Default value: 500
   * `args.compression_level`: (Optional) The GZIP compression level for HTTP(s) requests towards {ls}. It can be any integer value between 1 (minimum compression, best performance, highest amount of bytes sent) and 9 (maximum compression, worst performance, lowest amount of bytes sent). Default value: 1
-  * `args.ssl_assert_fingerprint`: (Optional) SSL fingerprint for self-signed SSL certificate on HTTPS transport.
+  * `args.ssl_assert_fingerprint`: (Optional) SSL fingerprint for self-signed SSL certificate on HTTPS transport. The default value is an empty string, meaning the HTTP client requires a valid certificate. 
```

Comment:
NIT: I wonder if we should explain what "valid" means here

Reply:
Good point. 

What about this updated description?

> SSL fingerprint for self-signed SSL/TLS certificate on HTTPS transport. The default value is an empty string, meaning the HTTP client requires a valid certificate (issued by a trusted CA, valid date range, matching CN, and valid certificate chain).

Reply:
That is definitely enough :)